shared/product.go

```diff
@@ -91,11 +95,36 @@ func ManageService(product, action, nodeType string, ips []string) (string, erro
 func CertRotate(product string, ips []string) (string, error) {
```

Comment:
also here, i know thats not the context now, but we are not validating if by any chance we receive ips empty